ledger/lruaccts.go

```diff
@@ -66,11 +78,29 @@ func (m *lruAccounts) flushPendingWrites() {
 		m.log.Warnf("lruAccounts: number of entries in pendingAccounts(%d) exceed the warning threshold of %d", pendingEntriesCount, m.pendingWritesWarnThreshold)
 	}
 	for ; pendingEntriesCount > 0; pendingEntriesCount-- {
+		done := false
```

Comment:
why do we need this? I guess the assumption is to have `pendingEntriesCount` synchronized with the `m.pendingAccounts` chan since they accessed under the `au.accountsMu` lock

Reply:
That's to break out of the for loop inside a switch statement.
The other alternative in Go is to use a label on the for loop and do `break label_name`.

Reply:
there is already some code that does `break outer`. I think this is shorter => better

Reply:
changed it to use labels.